src/Components/PropertyInspector/StandalonePropertyInspector.tsx

```diff
@@ -185,6 +204,9 @@ const StandalonePropertyInspector: React.FC<StandalonePropertyInspectorProps> =
                         onNodeValueUnset={onNodeValueUnset}
                         onAddMapValue={onAddMapValue}
                         onRemoveMapValue={onRemoveMapValue}
+                        onRemoveArrayItem={onRemoveArrayItem}
```

Comment:
it's a bummer that we have to duplicate all the callbacks just to add a type. Presumably it's a lot of surgery to simply pass back a type field on the existing callbacks and rename them be generic so it would scale to the next type we decide to add?

Reply:
Addressed comments and made some additional div-->IconButton migrations.